.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,16 @@
 <!-- DO NOT MODIFY OR DELETE THIS TEMPLATE. IT IS USED IN AUTOMATION. -->
-## Bug
 
-<!-- Search https://github.com/NuGet/Home/issues, and create one if you can't find a suitable issue. -->
-<!-- Paste the full link, like https://github.com/nuget/home/issues/1000. GitHub will render is neatly. -->
-Fixes:
+# Bug
 
-Regression? Last working version:
+<!-- If this is an engineering change or test change only, you do not need an issue. -->
+<!-- Find or create an issue in NuGet/Home and paste the full url. -->
+<!-- At the maintainers discretion, multiple changes may apply to a single issue, but only when the PRs are all created within a short period of time. -->
+Fixes:
```

Comment:
I know we have a `.editorconfig` setting to tells the editor to remove trailing whitespace, but can you override it for just this line, or maybe exclude this file from editorconfig?  There's been more than once I forgot to add a space after the `fixes:`, and then github didn't auto close the PR.

Reply:
I'll add that.